hyperspy/_signals/eels.py

```diff
@@ -287,12 +287,13 @@ def estimate_zero_loss_peak_centre(s, mask, signal_range):
                 else self.axes_manager[-1].axis[0])
         right = (right if right < self.axes_manager[-1].axis[-1]
                  else self.axes_manager[-1].axis[-1])
-        self.align1D(
-            left,
-            right,
-            also_align=also_align,
-            show_progressbar=show_progressbar,
-            **kwargs)
+        if self.axes_manager.navigation_size > 1:
+            self.align1D(
+                left,
+                right,
+                also_align=also_align,
+                show_progressbar=show_progressbar,
+                **kwargs)
```

Comment:
What about adding `else` write to the log that nothing was done because you can't align one single spectrum?